src/dev-guide/database-architecture.md

```diff
@@ -0,0 +1,28 @@
+# Database
+
+
+The architecture for the database can be seen in the following
+UML diagram:
+
+![](file:///C:/Users/Judy/AppData/Local/Temp/msohtmlclip1/01/clip_image002.png)![1666253938929](image/database-architecture/1666253938929.png)
```

Comment:
Not sure what's the deal with `clip_image002.png`, maybe it got carried away with whatever chinese markdown editor you're using?

Also, descriptive filenames and alt text!!!